NEXT-10854 support custom cache folders (#152)
shopwareBoostDay/platform: changelog/_unreleased/2020-09-17-next-10854-support-custom-cache-folders.md

> ---

Reviewer: No need to mention the ticket in the changelog filename

Author: file renamed, will remind that for the next pr

> title: NEXT-10854

Reviewer 2: ```suggestion
title: Support custom cache folders
```

Author: done, thanks for the hint

issue: NEXT-10854
author: Yann Karl
author_email: y.karl@webweit.de 

> author_github: yannick80

Reviewer: ```suggestion
author_github: @yannick80
```

Author: done, thanks for the hint

---
# Core

> *  Renamed `cacheDir` to `projectDir`, updated write path for `dump` in `src/Core/Framework/Plugin/BundleConfigDumper.php`

Reviewer: Please use `Added`, `Changed`, `Removed` or `Deprecated` as keywords here.

```suggestion
*  Changed property name `cacheDir` to `projectDir`, updated write path for `dump` in `src/Core/Framework/Plugin/BundleConfigDumper.php`
```

Author: also done, good to know for the next pr
